Microsoft.Research/Clousot.Cache/SQLCacheModel.cs

```diff
@@ -784,5 +786,33 @@ protected override SqlConnectionStringBuilder BuildConnectionString(IClousotCach
 
       return base.BuildConnectionString(options);
     }
+
+    public static string GetDesiredDbName()
```

Comment:
No, I did not change the behavior of any client (except that it does no longer fail).
There has been always only one instance of ```LocalDbClousotCacheFactory```, and now it will connect to the correct DB. 
We don't need to be backward compatible, the file names are different for every version anyhow.